container-build-task/Test/gcloud-fails.ts

```diff
@@ -17,14 +17,12 @@
  *   a failure in gcloud.
  * @author przybjw@google.com (Jim Przybylinski)
  */
+import {getDefaultAnswers, registerCommonMocks} from 'common/register-mocks';
 import * as path from 'path';
-import * as fs from 'fs';
-import { TaskLibAnswers } from 'vsts-task-lib/mock-answer';
-import { TaskMockRunner } from 'vsts-task-lib/mock-run';
-import {
-  registerCommonMocks, getDefaultAnswers, getFsMock
-} from 'common/register-mocks';
-import { failResult } from './test-constants'
+import {TaskLibAnswers} from 'vsts-task-lib/mock-answer';
+import {TaskMockRunner} from 'vsts-task-lib/mock-run';
+
```

Comment:
It was a style decision to separate external imports from module imports. I don't think there is a strong reason to remove it.